### PR TITLE
Add jf version and commit to healthcheck

### PR DESCRIPTION
**Pull request by roznawsk**
## Acknowledging the stipulations set forth:
 - [x] I hereby confirm that a Pull Request involving updates to the Software Development Kit (SDK) has been smoothly merged, currently awaits processing, or is otherwise deemed unnecessary in this context.
 - [x] I also affirm that another Pull Request, specifically addressing updates to the documentation body (commonly referred to as 'docs'), has either been successfully incorporated, is in the process of review, or is considered superfluous under the prevailing circumstances.

### CODE REVIEW

File: test/jellyfish_web/controllers/healthcheck_controller_test.exs [hunk 1]
```diff
@@ -5,6 +5,8 @@ defmodule JellyfishWeb.HealthcheckControllerTest do
 
   @schema JellyfishWeb.ApiSpec.spec()
 
+  @commit_hash_length 40
+
```
Comment (sgfn) [inline test/jellyfish_web/controllers/healthcheck_controller_test.exs:9]: Do we really need all 40 chars of the commit hash?

File: lib/jellyfish_web/api_spec/health_report.ex [hunk 2]
```diff
@@ -50,8 +50,10 @@ defmodule JellyfishWeb.ApiSpec.HealthReport do
     properties: %{
       status: Status,
       uptime: %Schema{type: :integer, description: "Uptime of Jellyfish (in seconds)"},
-      distribution: Distribution
+      distribution: Distribution,
+      version: %Schema{type: :string, description: "Version of Jellyfish"},
+      git_commit: %Schema{type: :string, description: "Commit hash of the build"}
```
Comment (sgfn) [inline lib/jellyfish_web/api_spec/health_report.ex:55]: This should be in camelCase

File: lib/jellyfish_web/api_spec/health_report.ex [hunk 3]
```diff
@@ -50,8 +50,10 @@ defmodule JellyfishWeb.ApiSpec.HealthReport do
...
     },
-    required: [:status, :uptime, :distribution]
+    required: [:status, :uptime, :distribution, :version]
```
Comment (Rados13) [inline lib/jellyfish_web/api_spec/health_report.ex:57]: Why `git_commit` is not required?

File: lib/jellyfish/config_reader.ex [hunk 4]
```diff
@@ -180,6 +180,16 @@ defmodule Jellyfish.ConfigReader do
     end
   end
 
+  def read_git_commit() do
+    System.get_env("JF_GIT_COMMIT") ||
+      with {_path, 0} <- System.cmd("which", ["git"]),
+           {commit, 0} <- System.cmd("git", ["rev-parse", "HEAD"]) do
+        commit |> String.trim()
+      else
+        _error -> nil
```
Comment (Rados13) [inline lib/jellyfish/config_reader.ex:189]: I think we should provide a default value here, e.g., "dev," or raise an exception; nil doesn't sound like a good value.

File: lib/jellyfish_web/controllers/healthcheck_controller.ex [hunk 5]
```diff
@@ -4,6 +4,8 @@ defmodule JellyfishWeb.HealthcheckController do
 
   alias JellyfishWeb.ApiSpec
 
+  @version Mix.Project.config()[:version]
+
```
Comment (sgfn) [inline lib/jellyfish_web/controllers/healthcheck_controller.ex:8]: [nitpick] We're adding this module attribute in more and more places -- you could consider putting it somewhere like `jellyfish.ex`, i.e. `def version(), do: @version`, then use `Jellyfish.version()`

**Review by sgfn — APPROVED** [review]

**Review by Rados13 — APPROVED** [review]

**Pull request merged**